schema/IN_KIND_OUT.json

```diff
@@ -120,15 +119,20 @@
         "back_reference_sched_name": {
             "title": "BACK REFERENCE SCHED NAME",
             "description": "",
-            "const": "SA11AI",
+            "type": "string",
+            "enum": [
+                "SA11AI",
+                "SA11AII"
```

Comment:
We never use SA11AII.  It's a concept that doesn't really fit our paradigm.  it's the legacy way of saying the transaction is unitemized.  No transaction is ever sent to the FEC with SA11AII because that's the definition.  As far as summing it up on the dsp, we handle that with the itemized field and look for SA11AII.  you can see that there examples of other SA11AI's where we have taken the SA11AII's out of the schema

Reply:
In light of Aurelia's comments at the requirements meeting today, let's hold onto keeping SA11AII in the schema as, while it doesn't affect our app, it is a general requirement.